api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirObservationServiceImpl.java

```diff
@@ -31,10 +32,11 @@
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
+@Getter(AccessLevel.PROTECTED)
 @Component
 @Transactional
 @Setter(AccessLevel.PACKAGE)
-public class FhirObservationServiceImpl implements FhirObservationService {
+public class FhirObservationServiceImpl extends BaseFhirService<Observation, org.openmrs.Obs> implements FhirObservationService {
```

Comment:
Remember that you need to rename the `observationTranslator` field to `translator`

Reply:
Am not getting you by here renaming ObservationTranslator to translator

Reply:
The field called `observationTranslator` in this class should be named `translator`.